tests/test/ported_libs/test_ffmpeg.cpp

```diff
@@ -58,4 +58,11 @@ TEST_CASE_METHOD(FFmpegTestFixture,
     faabric::Message msg = faabric::util::messageFactory("ffmpeg", "check");
     execFunction(msg);
 }
+
+TEST_CASE_METHOD(FFmpegTestFixture,
+                 "Test executing FFmpeg checks in WAMR",
+                 "[libs]")
+{
+    executeWithWamrPool("ffmpeg", "check");
+}
```

Comment:
This shouldn't be the only test that exercises the WAMR filesystem syscalls, instead we can think about reusing the wasm functions already executed using the WAVM-based version in `tests/test/faaslet/test_filesystem.cpp`.